client/profile-wizard/steps/business-details/flows/selective-bundle/selective-extensions-bundle/index.js

```diff
@@ -189,12 +193,38 @@ export const SelectiveExtensionsBundle = ( {
 		};
 	} );
 
+	const { installedPlugins = [] } = useSelect( ( select ) => {
+		const { getInstalledPlugins } = select( PLUGINS_STORE_NAME );
+
+		return {
+			installedPlugins: getInstalledPlugins(),
+		};
+	} );
```

Comment:
This shouldn't be necessary as the free extensions list includes a `is_installed` prop, we should just filter by `is_installed` anyway. Or `is_activated` if we want to be more specific (you could add the `is_activated` on the back-end for the free features request in that case.

Reply:
Good idea! I addressed these changes in the commits 25f82f7 and 43fc2d